index.js

```diff
@@ -4,3 +4,70 @@ console.log(square.style.left, square.style.top);
 document.addEventListener("keydown", function(e) {
   console.log(e.key)
 });
+
+let body = document.querySelector('body');
+
+document.addEventListener("keydown", function(e){
+  console.log(e);
+  if(e.key === 'ArrowLeft' && square.style.left !== "0px"){
+    let leftPixel = parseInt(square.style.left);
+    leftPixel -= 10;
+    square.style.left = leftPixel + "px";
+  }
+  if(e.key === 'ArrowRight' && square.style.left !== "300px"){
+    let rightPixel = parseInt(square.style.left);
+    rightPixel += 10;
+    square.style.left = rightPixel + "px";
+  }
+  if(e.key === 'ArrowDown' && square.style.top !== "300px"){
+    let downPixel = parseInt(square.style.top);
+    downPixel += 10;
+    square.style.top = downPixel + "px";
+  }
+  if(e.key === 'ArrowUp' && square.style.top !== "0px"){
+    let upPixel = parseInt(square.style.top);
+    upPixel -= 10;
+    square.style.top = upPixel + "px";
+  }
+    
+})
+function randColo(){
+  let chars = '0123456789ABCDEF';
+  let color = '#';
+  for(let i = 0; i < 6; i++){
+    color += chars[Math.floor(Math.random() * 16)];
+  }
+  return color;
+}
```

Comment:
This is a clever way to generate a random color! Here is a [one-line trick](https://css-tricks.com/snippets/javascript/random-hex-color/) to create any random color.